topydo/lib/PrettyPrinterFilter.py

```diff
@@ -117,3 +145,119 @@ def filter(self, p_todo_str, _):
                 p_todo_str)
 
         return p_todo_str
+
+
+class PrettyPrinterBasicPriorityFilter(PrettyPrinterFilter):
+
+    """ Adds strips the bracked from the priority of the todo item. Add a
+        (blank) space if not priority is specified (to maintain printing
+        alignment)
+    """
+
+    def __init__(self, p_replacement=" "):
+        super(PrettyPrinterBasicPriorityFilter, self).__init__()
+        self.p_replacement = p_replacement
+
+    def filter(self, p_todo_str, _):
+        """ Find the priority """
+        matches = re.search('^(?P<id>\| *\w+\| )?(\((?P<pri>[A-Z])\) )?', p_todo_str)
```

Comment:
Please add a comment that gives an example of a string you're trying to match. That makes parsing this regex a bit easier.